.github/workflows/deploy-theme.yaml

```diff
@@ -0,0 +1,15 @@
+name: Deploy theme
+on:
+  push:
+    branches:
+      - feat/iota-blog
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Deploy Ghost Theme
+        uses: TryGhost/action-deploy-theme@v1.6.1
```

Comment:
I think i prefer not using this action for now, this is to upload directly the theme to a server, but I think we are safer just creating the artifact and be able to download it from the gh action panel. in localhost I use `yarn production`, can we do something similar to what localhost does just to generate the zip?